Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
+        return self.field(field)
+    }
+}
+
+
+struct Artist {
+    let id: Int?
+    // Note the optionality. Fields that are marked `NOT NULL` in your schema should be non-optional.
+    var name: String
+    var genre: String?
+
+    // You probably want your own initializer
+    init(name: String, genre: String) {
+        self.id = nil
+        self.name = name
+        self.genre = genre
+    }
+}
+
+
+extension Artist: Table {
```

Comment:
I kinda think the opposite. I'd like to understand more about `Model` vs `Table` as well. IMO I think, since we're talking about `SQL`, having a `Table` type makes more sense than having a `Model` type. Models are domain abstractions, and I think they should be separate from db concepts.